apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -93,24 +94,18 @@ const FreeResponseAIEvaluation: React.FunctionComponent<
         isPending={evaluationsPending}
         studentWorkEvaluations={evaluations}
         evaluationComplete={evaluationComplete}
+        totalStudentCount={responses.length}
+        openDetailedAnalysis={() => setShowDetailedAnalysis(true)}
       />
-      {evaluationComplete && aiSummary && (
+      {evaluationComplete && aiSummary && showDetailedAnalysis && (
         <div>
-          <CollapsibleSection
-            headerContent={
-              <h3>AI Evaluations of Individual Student Responses</h3>
-            }
-          >
-            <div>
-              {evaluations.map(evaluation => (
-                <FreeResponseStudentResponseRow
-                  key={evaluation.studentId}
-                  studentResponse={evaluation}
-                  studentWorkEvaluation={evaluation}
-                />
-              ))}
-            </div>
-          </CollapsibleSection>
+          {evaluations.map(evaluation => (
+            <FreeResponseStudentResponseRow
+              key={evaluation.studentId}
+              studentResponse={evaluation}
+              studentWorkEvaluation={evaluation}
```

Comment:
do you need both of these if they are passing in the same information?

Reply:
Good point - removed one!